### PR TITLE
Adjusted instructions on how to use the client

### PR DESCRIPTION
**Pull request by tzengerink**
This adds an (over) simplified example on how to use the client. Since this will also end up on [the NPM package page](https://www.npmjs.com/package/usabilla-api), this will allow the user to get started without the need for actually looking at the source code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +21,23 @@ Install node client through npm
...
-how the client can be used and what is possible.
+### Usage
 
-```bash
-$ node example/example.js "your_access_key" "your_secret_key"
+An example that displays the number of buttons:
+
+```js
+var Usabilla = require('usabilla-api');
+var api = new Usabilla('YOUR-ACCESS-KEY', 'YOUR-SECRET-KEY');
```
Comment (spirosikmd) [inline README.md]: I would just change that to `var usabilla = new Usabilla(...)`. Looks better 😊

File: README.md [hunk 2]
```diff
@@ -21,13 +21,23 @@ Install node client through npm
...
+
+api.websites.buttons.get().then((buttons) => {
+    console.log('Number of buttons: ', buttons.length);
+}).catch((reason) => {
+    console.error(reason);
+});
 ```
 
+See the code in the 'example' folder for more advanced ideas on how to use the client.
```
Comment (spirosikmd) [inline README.md]: You can use a relative link here for `example` folder.

`[example](example)`

Comment (tzengerink) [inline README.md]: Will that also work on the [NPM package page](https://www.npmjs.com/package/usabilla-api)?

Comment (tzengerink) [inline README.md]: Nevermind, ofcourse it will.

**Pull request merged**